features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumScopeKey.kt

```diff
@@ -0,0 +1,92 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.internal.domain.scope
+
+import android.app.Activity
+import android.content.ComponentName
+import androidx.navigation.ActivityNavigator
+import androidx.navigation.fragment.DialogFragmentNavigator
+import androidx.navigation.fragment.FragmentNavigator
+
+/**
+ * Uniquely identifier to link multiple events relating to the same entity.
+ * This fixes an issue where we would keep (weak) references to activities/fragments
+ *
+ */
+internal data class RumScopeKey(
+    val id: String,
+    val url: String,
+    val name: String?
```

Comment:
It seems that this property is not used?

Reply:
Missed an additional commit